ateam_ui/install_deps.sh

```diff
@@ -6,10 +6,22 @@
 
 set -e
 
+if ! command -v npm; then
+  echo "ERROR: npm is not installed"
+
+  echo "attempting to install nodejs and npm"
+  sudo apt update -y
+  sudo apt install nodejs npm -y
+  if ! command -v npm; then
+    echo "failed to install npm"
+    exit 1
+  fi
+fi
```

Comment:
npm should have been installed by rosdep (because it's listed as a dependency in the package manifest for ateam_ui)
nodejs is a dependency of npm, so it also should've been installed in the rosdep step.

Reply:
this is definitely because I tried to skip ahead to see if it would help resolve dependencies